apps/src/javalab/JavabuilderConnection.js

```diff
@@ -20,7 +21,8 @@ export default class JavabuilderConnection {
       data: {
         projectUrl: dashboard.project.getProjectSourcesUrl(),
         channelId: this.channelId,
-        projectVersion: dashboard.project.getCurrentSourceVersionId()
+        projectVersion: dashboard.project.getCurrentSourceVersionId(),
+        levelId: this.levelId
       }
     })
       .done(result => this.establishWebsocketConnection(result.token))
```

Comment:
Will the level also need to be added to the localhost url parameters (line 42 in the new code - github just wouldn't let me comment there)

Reply:
oh yep good call